src/senselab/__init__.py

```diff
@@ -1 +1,22 @@
 """.. include:: ../../README.md"""  # noqa: D415
+
+from typing import Iterator
+
+import nest_asyncio
+import numpy as np
+import torch
+from pydra.utils.hash import Cache, bytes_repr_sequence_contents, register_serializer
+
+nest_asyncio.apply()
+
+
+@register_serializer(torch.Tensor)
```

Comment:
hey @wilke0818 why is the `bytes_repr_arraylike` both here and in the `pydra_helpers.py`?

Reply:
well that's part of the question I had: where do we want this to live? If we leave it in `pydra_helpers.py` then we need to import it in the `init`. I wasn't sure which made more sense especially as I am not sure if we will have more pydra related utilities? You can kind of see the notes I was making for what a Pydra class could look like.

Reply:
i like the idea of having it in pydra_helpers and import it in the init. we will probably need something similar for setting up global settings for huggingface and other libraries at some point. just make a decision and let's see how it works for now. we can always change if we realize that there are better options

Reply:
done. we should look at using \_\_all__ though I don't fully understand it/would require a bit more restructuring than I was originally intending with this PR